Dockerfile

```
@@ -1,17 +1,18 @@
 FROM ubuntu:latest
 
 # Setup environment variables for Docker image
-ENV FC=/usr/bin/mpif90 CC=/usr/bin/mpicc CXX=/usr/bin/mpicxx \
+ENV CC=/usr/bin/mpicc CXX=/usr/bin/mpicxx \
     PATH=/opt/openmc/bin:/opt/NJOY2016/build:$PATH \
     LD_LIBRARY_PATH=/opt/openmc/lib:$LD_LIBRARY_PATH \
     OPENMC_CROSS_SECTIONS=/root/nndc_hdf5/cross_sections.xml \
-    OPENMC_ENDF_DATA=/root/endf-b-vii.1
+    OPENMC_ENDF_DATA=/root/endf-b-vii.1 \
```

```suggestion
    OPENMC_ENDF_DATA=$HOME/endf-b-vii.1 \
```
We extract tar archive to  `tar -C $HOME -xJ`.